### PR TITLE
Easier l10n

### PR DESCRIPTION
**Pull request by mythmon**
Talking with @mozillamarcia in `#buddyup`, I realized a couple issues with the localization. This fixes those.

r?

### CODE REVIEW

File: tasks/get_localization.js [hunk 1]
```diff
@@ -25,24 +25,18 @@ module.exports = function(grunt) {
...
-        if (ignorePatterns[i].exec(name)) {
-          return false;
-        }
-      }
-      return true;
-    })
-    .map(function(name) {
-      return name.replace('_', '-');
-    });
+    var locales = [
```
Comment (rehandalal) [inline tasks/get_localization.js]: i guess the one annoying thing about this is we have to remember to maintain this list

Comment (mythmon) [inline tasks/get_localization.js]: Yeah, which isn't great. Do you know another place we could get this list? Maybe we need to make an API in Sumo for it?

Comment (rehandalal) [inline tasks/get_localization.js]: we should file a bug for the api

**Comment by rehandalal** [comment]
wfm r+

**Pull request merged**